### PR TITLE
Added process function for every file

### PR DESCRIPTION
**Pull request by Keyes**
This patch makes it possible to wrap each library for example in a self executing function with "use strict"

### CODE REVIEW

File: tasks/bower-concat.js [hunk 1]
```diff
@@ -57,6 +58,8 @@ module.exports = function(grunt) {
 		function concatenateAndWriteFile(files, destination, separator) {
 			if (!destination || !files || !files.length) return;
 
+			files = _.map(files, process);
```
Comment (sapegin) [inline tasks/bower-concat.js]: What if process is undefined?

**Comment by sapegin** [comment]
Thanks! Could you please add some docs too?

File: tasks/bower-concat.js (hunk 1 above)
Comment (Keyes) [inline tasks/bower-concat.js]: then nothing happens and the files will not be changed

**Comment by Keyes** [comment]
Updated Readme.md with new process option

**Pull request merged**